<?xml version="1.0"?>
<launch>
    <arg name="topic" default="/webcam/image_raw" />
    <arg name="show_image_frame" default="false" />
    <arg name="cascade_file" default="$(find face_detection)/config/cascade.xml" />
    <arg name="video_file" default="" />
    <arg name="bdf_file" default="" />
    <node pkg="legacy_measurement" type="legacy_measurement.py" name="legacy_measurement" output="screen" cwd="node">
        <param name="topic" type="string" value="$(arg topic)" />
        <param name="show_image_frame" type="bool" value="$(arg show_image_frame)" />
        <param name="cascade_file" type="string" value="$(arg cascade_file)" />
        <param name="video_file" type="string" value="$(arg video_file)" />
        <param name="bdf_file" type="string" value="$(arg bdf_file)" />
    </node>
</launch>
